.pre-commit-config.yaml

```diff
@@ -63,6 +63,8 @@ repos:
     rev: v2.15.5
     hooks:
     -   id: pylint
+        language: system
+        stages: [manual]
```

Comment:
What are the implications of these changes?

Reply:
it'll run like the `mypy` one, only in CI